### PR TITLE
Salem 10

### PR DESCRIPTION
**Pull request by salemdaniel007**

### CODE REVIEW

**Comment by salemdaniel007** [comment]
gkhhkh

File: src/App.vue [hunk 1]
```diff
@@ -11,7 +11,7 @@
         <li>
           <router-link to="/history" class="navtext"
             ><i class="fa fa-history fa-3x" style="color:white;"></i>
-            <span>History</span>
+            <span>Historyyy</span>
```
Comment (salemdaniel007) [inline src/App.vue:14]: this. and that

Comment (salemdaniel007) [reply src/App.vue:14]: yeah yeah

File: src/views/History.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="history">
-    <div><h1>BEHOLD... the works of <span>thy hands</span></h1></div>
```
Comment (salemdaniel007) [inline src/views/History.vue:3]: something light

Comment (salemdaniel007) [reply src/views/History.vue:3]: yo

**Comment by salemdaniel007** [comment]
Leave a comment

**Comment by salemdaniel007** [comment]
Leave a comment

**Comment by second-son** [comment]
Leave a comment

**Comment by second-son** [comment]
Leave a comment

File: src/App.vue (hunk 1 above)
Comment (salemdaniel007) [reply src/App.vue:14]: Leave a comment

Comment (salemdaniel007) [reply src/App.vue:14]: I am leaving a comment here

Comment (salemdaniel007) [reply src/App.vue:14]: I am leaving a comment

Comment (second-son) [reply src/App.vue:14]: leaving my own too

Comment (salemdaniel007) [reply src/App.vue:14]: i am enjoying this

Comment (salemdaniel007) [reply src/App.vue:14]: so this is what the dm thing looks like

Comment (second-son) [reply src/App.vue:14]: as in
shocking abi

Comment (salemdaniel007) [reply src/App.vue:14]: Let us do this again

Comment (salemdaniel007) [reply src/App.vue:14]: What do we have here

Comment (second-son) [reply src/App.vue:14]: i don't know shenzy

Comment (salemdaniel007) [reply src/App.vue:14]: let us do this again

Comment (salemdaniel007) [reply src/App.vue:14]: who be this?

Comment (salemdaniel007) [reply src/App.vue:14]: alright

Comment (salemdaniel007) [reply src/App.vue:14]: this is a test

Comment (salemdaniel007) [reply src/App.vue:14]: again

Comment (salemdaniel007) [reply src/App.vue:14]: testing again

Comment (second-son) [reply src/App.vue:14]: let us see if the logic holds up

Comment (salemdaniel007) [reply src/App.vue:14]: apparently it does

Comment (salemdaniel007) [reply src/App.vue:14]: checking the url

Comment (second-son) [reply src/App.vue:14]: one mans food na another man's poison

Comment (salemdaniel007) [reply src/App.vue:14]: not bad

File: src/views/History.vue (hunk 2 above)
Comment (second-son) [reply src/views/History.vue:3]: yoga

Comment (second-son) [reply src/views/History.vue:3]: sennd

Comment (second-son) [reply src/views/History.vue:3]: here

Comment (second-son) [reply src/views/History.vue:3]: here2

**Comment by salemdaniel007** [comment]
ggg

File: src/views/History.vue (hunk 2 above)
Comment (salemdaniel007) [reply src/views/History.vue:3]: tyui

Comment (second-son) [reply src/views/History.vue:3]: iiojghj

**Review by second-son — COMMENTED** [review]
nnnn

File: src/views/History.vue (hunk 2 above)
Comment (second-son) [reply src/views/History.vue:3]: balablu

**Review by second-son — COMMENTED** [review]
bulaba

File: src/views/History.vue (hunk 2 above)
Comment (second-son) [reply src/views/History.vue:3]: i said balablu

**Review by second-son — COMMENTED** [review]
bulaba

File: src/views/History.vue (hunk 2 above)
Comment (second-son) [reply src/views/History.vue:3]: bulabaaaa

**Review by second-son — COMMENTED** [review]
balablu!

File: src/views/Home.vue [hunk 3]
```diff
@@ -6,7 +6,7 @@
         <input @change="getQrCode" type="text" placeholder="Enter a valid url" v-model="url" />
       </div>
       <h5 style="color:#f15c12;" class="tries" v-show="count > 10">
-        Err.. boss man, calm down first<span style="font-size: 1.5rem;">💆🏿‍♂️</span>, you don
+        Boss man, calm down first<span style="font-size: 1.5rem;">💆🏿‍♂️</span>, you don
```
Comment (odetolataiwo) [inline src/views/Home.vue:9]: I no wan calm down. Pay me my money

File: src/App.vue [hunk 4]
```diff
@@ -11,7 +11,7 @@
         <li>
           <router-link to="/history" class="navtext"
             ><i class="fa fa-history fa-3x" style="color:white;"></i>
```
Comment (odetolataiwo) [inline src/App.vue:13]: random comment

File: src/App.vue [hunk 5]
```diff
@@ -11,7 +11,7 @@
         <li>
```
Comment (odetolataiwo) [inline src/App.vue:11]: radom

File: src/App.vue (hunk 4 above)
Comment (second-son) [reply src/App.vue:13]: another random comment

File: src/App.vue [hunk 6]
```diff
@@ -11,7 +11,7 @@
...
           <router-link to="/history" class="navtext"
             ><i class="fa fa-history fa-3x" style="color:white;"></i>
-            <span>History</span>
+            <span>Historyyy</span>
           </router-link>
         </li>
```
Comment (second-son) [inline src/App.vue:16]: lee mao

Comment (second-son) [reply src/App.vue:16]: a man has no review

Comment (second-son) [reply src/App.vue:16]: A man has said a man has no review

Comment (second-son) [reply src/App.vue:16]: A man has said a man has no review

Comment (second-son) [reply src/App.vue:16]: app.client.chat.postMessage(

Comment (second-son) [reply src/App.vue:16]: e be like ice water

Comment (second-son) [reply src/App.vue:16]: e be like ice water

Comment (second-son) [reply src/App.vue:16]: i am trying to understand what is going on here, why is it not removing me as a reviewer again biko?

Comment (second-son) [reply src/App.vue:16]: foo bar

**Review by second-son — COMMENTED** [review]
leave a comment

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: meow

Comment (second-son) [reply src/App.vue:16]: meow

**Review by second-son — COMMENTED** [review]
meow again

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: another meow

**Review by second-son — COMMENTED** [review]
another meow again

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: testing the meow

**Review by second-son — COMMENTED** [review]
another testing the meow

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: oh.... meeoooow....

**Review by second-son — COMMENTED** [review]
what a meeoooow....

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: 2 + 5 is equal to 7

**Review by second-son — COMMENTED** [review]
1 + 6 is equal to 7

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: Ma review has been requested

**Review by second-son — COMMENTED** [review]
WHo requested ma review?

**Review by second-son — COMMENTED** [review]
LGTM

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: HAAAA

**Review by second-son — COMMENTED** [review]
BAAAA

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: Ma review has been requestedDDDD

**Review by second-son — COMMENTED** [review]
Ma review

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: BULABA

**Review by second-son — COMMENTED** [review]
BALABLU

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: GHJ

**Review by second-son — COMMENTED** [review]
(_review submitted_)

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: yababuluku

**Review by second-son — CHANGES_REQUESTED** [review]
bulaba

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: aiina

**Review by second-son — APPROVED** [review]
noina

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: base on believe

**Review by second-son — COMMENTED** [review]
BASE ON BELIEVE

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: base on believe

**Review by second-son — COMMENTED** [review]
BASE ON BELIEVE

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: loading

**Review by second-son — COMMENTED** [review]
LOADING

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: console.log("COMMENT CREATED 🚨")
console.log(res.mapped_members)

**Review by second-son — COMMENTED** [review]
LOADING

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: console.log("COMMENT CREATED 🚨")
console.log(res.mapped_members)

**Review by second-son — COMMENTED** [review]
A man's review

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: console.log("COMMENT CREATED 🚨")
console.log(res.mapped_members)

**Review by second-son — COMMENTED** [review]
A man has a review

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: console.log("COMMENT CREATED 🚨")
console.log(res.mapped_members)

**Review by second-son — COMMENTED** [review]
console.log("COMMENT CREATED 🚨")
console.log(res.mapped_members)

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: console.log("COMMENT CREATED 🚨")
console.log(res.mapped_members)

**Review by second-son — COMMENTED** [review]
console.log("COMMENT CREATED 🚨")
console.log(res.mapped_members)

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: console.log("COMMENT CREATED 🚨")
console.log(res.mapped_members)

**Review by second-son — COMMENTED** [review]
console.log("COMMENT CREATED 🚨")
console.log(res.mapped_members)

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
          app,
          pullRequestPayload,
          message,
          assigneeSlackUserId
        );

**Review by second-son — COMMENTED** [review]
sendDMMessageNoAttachment(
          app,
          pullRequestPayload,
          message,
          assigneeSlackUserId
        );

File: src/App.vue (hunk 6 above)
Comment (second-son) [reply src/App.vue:16]: console.log("COMMENT CREATED 🚨")
console.log(res.mapped_members)

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: amazing grace

Comment (second-son) [reply src/App.vue:16]: how sweet the sound

Comment (second-son) [reply src/App.vue:16]: that saved a wretch

Comment (second-son) [reply src/App.vue:16]: like me

Comment (second-son) [reply src/App.vue:16]: but for your grace

Comment (second-son) [reply src/App.vue:16]: that saved a wretch

Comment (second-son) [reply src/App.vue:16]: Amazing grace

Comment (second-son) [reply src/App.vue:16]: forever grateful

Comment (second-son) [reply src/App.vue:16]: but for your grace

Comment (second-son) [reply src/App.vue:16]: for your amazing grace

Comment (second-son) [reply src/App.vue:16]: debounce

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: kjkubiiybihbiyb.  on

Comment (second-son) [reply src/App.vue:16]: lkhjghcbvhkjghf b

Comment (second-son) [reply src/App.vue:16]: jrjr45553

Comment (second-son) [reply src/App.vue:16]: lkkibuu8787guybjhb.  kjkj

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: hgjkghjkhgjhk

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: you have been faithful

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: this is a preview of how this would look

Comment (second-son) [reply src/App.vue:16]: sendDMMessageNoAttachment(
app,
pullRequestPayload,
message,
assigneeSlackUserId
);

Comment (second-son) [reply src/App.vue:16]: this is a preview of how this would look

Comment (salemdaniel007) [reply src/App.vue:16]: Okat, let's take this for a spin and see

Comment (salemdaniel007) [reply src/App.vue:16]: we are taking this for a spin

Comment (salemdaniel007) [reply src/App.vue:16]: nothing as fun as taking stuff for a spin

Comment (salemdaniel007) [reply src/App.vue:16]: it's always cool to take stuff for a spin

Comment (salemdaniel007) [reply src/App.vue:16]: spinning spinning, we are spinning, are we spinning? yes.... mommy

Comment (salemdaniel007) [reply src/App.vue:16]: oya spin lemme see.. you

Comment (second-son) [reply src/App.vue:16]: spinning time

Comment (second-son) [reply src/App.vue:16]: some very random foo bar

Comment (salemdaniel007) [reply src/App.vue:16]: another very random foo bar

Comment (salemdaniel007) [reply src/App.vue:16]: very random foo bar

Comment (salemdaniel007) [reply src/App.vue:16]: very random foo bar

Comment (salemdaniel007) [reply src/App.vue:16]: random foo bar

Comment (salemdaniel007) [reply src/App.vue:16]: another random foo bar

Comment (second-son) [reply src/App.vue:16]: my own piece of foo bar

Comment (second-son) [reply src/App.vue:16]: my own piece of foo bar

Comment (second-son) [reply src/App.vue:16]: mirroring stuff

Comment (salemdaniel007) [reply src/App.vue:16]: my own piece of foo bar

Comment (second-son) [reply src/App.vue:16]: mirroring stuff again

Comment (second-son) [reply src/App.vue:16]: mirroring stuff again again

Comment (salemdaniel007) [reply src/App.vue:16]: my own piece of foo bar

**Pull request closed without merging**